tq/api.go

```diff
@@ -57,7 +58,7 @@ func (c *tqClient) Batch(remote string, bReq *batchRequest) (*BatchResponse, err
 	tracerx.Printf("api: batch %d files", len(bReq.Objects))
```

Comment:
@ttaylorr I've changed that line to:
```
 tracerx.Printf("api: batch %d files (retries %d)", len(bReq.Objects), c.MaxRetries)
```
.... and I still get `0` retries. Even with 7722c2f !

Reply:
@larsxschneider I am unable to reproduce this issue locally:

```
~/D/repo (master!) $ GIT_TRACE=1 git lfs push --all origin master
# ...
trace git-lfs: tq: sending batch of size 100
trace git-lfs: api: batch 100 files (8 retries)
```

Are you sure that you don't have any extra `lfs.transfer.maxretries` entries laying around anywhere?

Reply:
You are right. I see `trace git-lfs: api: batch 1 files (retries 10)` 👍